docs/src/index.md

```diff
@@ -120,7 +120,10 @@ integrated they are with DataFrames.jl.
       DataFrames.jl, other tabular data libraries (more on those below), and even
       non-tabular data. Provides many convenience functions analogous to those in
       dplyr in R or [LINQ](https://en.wikipedia.org/wiki/Language_Integrated_Query).
-    - You can find more on both of these packages in the
+    - [DataFrameMacros.jl](https://github.com/jkrumbiegel/DataFrameMacros.jl):
```

Comment:
Put this before Query.jl given that it's similar to DataFramesMeta?

Reply:
good point